src/components/BillStatusTag/BillStatusTag.svelte

```diff
@@ -0,0 +1,28 @@
+<script lang="ts">
+	import { BillStatus } from '$models/bill';
+	import { Tag } from 'carbon-components-svelte';
+
+	export let isLarge = false;
+	export let status: BillStatus;
+	let label = status;
+	let className = '';
+
+	switch (status) {
+		case BillStatus.InProgress:
+			className = 'bg-yellow-20 text-text-10';
```

Comment:
We shouldn't use `className` name here, that's the variable we got from props on line 25.